openfe_gromacs/protocols/gromacs_md/md_methods.py

```diff
@@ -654,3 +848,228 @@ def _execute(
             "generation": self._inputs["generation"],
             **outputs,
         }
+
+
+class GromacsMDRunUnit(gufe.ProtocolUnit):
+    """
+    Protocol unit for running plain MD simulations (NonTransformation)
+    in Gromacs.
+    """
+
+    def _run_gromacs(
+        self,
+        mdp,
+        in_gro,
+        top,
+        tpr,
+        out_gro,
+        xtc,
+        trr,
+        cpt,
+        log,
+        edr,
+        shared_basebath,
+    ):
+        assert os.path.exists(in_gro)
+        assert os.path.exists(top)
+        assert os.path.exists(mdp)
+        p = subprocess.Popen(
+            [
+                "gmx",
+                "grompp",
+                "-f",
+                mdp,
+                "-c",
+                in_gro,
+                "-p",
+                top,
+                "-o",
+                tpr,
+            ],
+            stdin=subprocess.PIPE,
+        )
+        p.wait()
+        assert os.path.exists(tpr)
+        p = subprocess.Popen(
+            [
+                "gmx",
+                "mdrun",
+                "-s",
+                tpr.name,
+                "-cpo",
+                cpt,
+                "-o",
+                trr,
+                "-x",
+                xtc,
+                "-c",
+                out_gro,
+                "-e",
+                edr,
+                "-g",
+                log,
+            ],
```

Comment:
may we pin `-ntmpi 1` and then let `-ntomp` be user defined.

Reply:
Changed this, defining `ntomp` in the simulation settings for now.